app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -391,4 +369,35 @@ open class FenixApplication : LocaleAwareApplication() {
         applicationContext.resources.configuration.uiMode = config.uiMode
         super.onConfigurationChanged(config)
     }
+
+    companion object {
```

Comment:
As for a companion object, I think that `FenixApplication` should not own `strictMode`. A personal file might help us be clearer in what our object does.  So maybe creating a `StrictModeManager` (or any other name that conveys what we're doing !) would help clear any confusion and allow other devs to know what we're doing with it 😄